addons/web/static/src/views/fields/properties/property_value.xml

```diff
@@ -92,6 +92,7 @@
                     </t>
                     <Many2XAutocomplete
                         t-else=""
+                        t-key="window._.uniqueId()"
```

Comment:
As discussed, see if crypto.randomUUID() can be used instead of the `_` (underscore) library